File: ePrijevozSarajevo/UI/eprijevoz_desktop/lib/models/country.dart
import 'package:json_annotation/json_annotation.dart';
part 'country.g.dart';

@JsonSerializable()
class Country {
  int? countryId;
  String? name;
  DateTime? modifiedDate;
  int? currentUserId;

  Country({this.countryId, this.name});

  factory Country.fromJson(Map<String, dynamic> json) =>
      _$CountryFromJson(json);

  Map<String, dynamic> toJson() => _$CountryToJson(this);
}

File: ePrijevozSarajevo/ePrijevozSarajevo.Services/Database/Country.cs
namespace ePrijevozSarajevo.Services.Database
{
    public class Country
    {
        public int CountryId { get; set; }
        public string Name { get; set; } = null!;
        public DateTime? ModifiedDate { get; set; }
        public int? CurrentUserId { get; set; }
    }
}
